cpu/atmega_common/avr_libc_extra/include/vendor/stdio.h

```diff
@@ -0,0 +1,1006 @@
+/* Copyright (c) 2002, 2005, 2007 Joerg Wunsch
```

Comment:
so this is not the default avr libs stdio.h?

Reply:
I took it from here https://www.nongnu.org/avr-libc/user-manual/stdio_8h_source.html